chia/util/config.py

```diff
@@ -162,3 +164,34 @@ def traverse_dict(d: Dict, key_path: str) -> Any:
         return val
     else:
         raise KeyError(f"value not found for key: {key}")
+
+
+start_methods: Dict[Optional[str], Optional[str]] = {
+    None: None,
+    "default": None,
+    "fork": "fork",
+    "forkserver": "forkserver",
+    "spawn": "spawn",
+}
+
+
+def process_config_start_method(
+    method: object,
```

Comment:
This is also confusing, why not just pass the config object?

Reply:
Passing in a thousand items when one is all that is needed is often not great.  I'll flip this around to your suggestion here.  The config needs a redesign anyways.